fenix/app/src/main/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckState.kt

```diff
@@ -29,11 +29,12 @@ sealed interface ReviewQualityCheckState : State {
      *
      * @property productReviewState The state of the product the user is browsing.
      * @property productRecommendationsPreference User preference whether to show product
-     * recommendations. True if product recommendations should be shown.
+     * recommendations. True if product recommendations should be shown. Null indicates that product
+     * recommendations are disabled.
      */
     data class OptedIn(
         val productReviewState: ProductReviewState = ProductReviewState.Loading,
-        val productRecommendationsPreference: Boolean,
+        val productRecommendationsPreference: Boolean?,
```

Comment:
optional nit: wondering aloud if `productRecommendationsEnabled` makes more sense for this being a boolean.

Reply:
It kinda does, but since we're using null as a feature disabled state and true/false as on/off instead of user enabled/user disabled, I think it's fine to keep the name as is.